public/app/features/dimensions/types.ts

```diff
@@ -66,7 +66,9 @@ export interface TextDimensionConfig extends BaseDimensionConfig<string> {
 }
 
 /** Use the color value from field configs */
-export interface ColorDimensionConfig extends BaseDimensionConfig<string> {}
+export interface ColorDimensionConfig extends BaseDimensionConfig<string> {
+  mode?: 'fixed';
```

Comment:
:thinking: -- the current logic use a field when it is set or the fixed color.

I don't think we want or need to add a new 'mode', but can have the editor have a flag that forces only fixed